promql/testdata/native_histograms.test

```diff
@@ -0,0 +1,207 @@
+# Minimal valid case: an empty histogram.
+load 5m
+	empty_histogram{}	{{}}
+
+eval instant at 5m histogram_count(empty_histogram{})
+	{} 0
+
+eval instant at 5m histogram_sum(empty_histogram{})
+	{} 0
+
+eval instant at 5m histogram_fraction(-Inf, +Inf, empty_histogram{})
+	{} NaN
+
+eval instant at 5m histogram_fraction(0, 8, empty_histogram{})
+	{} NaN
+
+
+
+# buckets:[1 2 1] means 1 observation in the 1st bucket, 2 observations in the 2nd and 1 observation in the 3rd (total 4).
+load 5m
+	single_histogram{}	{{schema:0 sum:4 count:4 buckets:[1 2 1]}}
```

Comment:
Maybe use a different value for `sum` to make sure we are seeing the right values below.

Reply:
Changed sum value and subsequent assertion in 3eb0735a7.